NOTICES.md

```diff
@@ -113,3 +113,9 @@ OpenSSL 1.1.1
 * License: https://github.com/openssl/openssl/blob/OpenSSL_1_1_1-stable/LICENSE
 * Project: https://www.openssl.org/
 * Source: https://github.com/openssl/openssl or https://github.com/ibmruntimes/openssl
```

Comment:
Why is this still here?

Reply:
I'm fine with deferring removal of references to version 1.x, but I don't think we can (or want to) make reference to the license of every version of OpenSSL someone *might* want to build with. I think we should expect those people to know what they're doing and understand the implications of building against some version and/or repository other than what OpenJ9 (advises and) uses.

I also think the comments
> If you built an OpenJDK with OpenJ9 that includes OpenSSL **v1.x** support ...

are (at best) redundant and should also be removed at the same time we update this file.

Reply:
It doesn't sound like you are asking me to change anything now, if that's not the case pls clarify. The diff between 1.x and 3.x is OpenSSL changed the license, otherwise we wouldn't have two entries.

Reply:
I won't object if you want to remove the references to 1.x now, but I'm not *asking* you to do so.

The only change I'm waiting for is to fix the typo mentioned in https://github.com/eclipse-openj9/openj9/pull/14900#discussion_r1304480338.